test/functional/test-viewport-binding.js

```diff
@@ -438,293 +438,292 @@ describes.realWin('ViewportBindingIosEmbedWrapper', {ampCss: true}, env => {
   });
 });
 
-describes.realWin('ViewportBindingIosEmbedShadowRoot_', {ampCss: true}, env => {
+describes.realWin('ViewportBindingIos', {ampCss: true}, env => {
   let iframe;
   let win;
   let binding;
   let vsync;
   let child;
 
-  beforeEach(function() {
-    if (!env.win.Element.prototype.attachShadow) {
-      // Can only test when SD is supported.
-      this.skip();
-    }
-    iframe = env.iframe;
-    iframe.style.width = '100px';
-    iframe.style.height = '100px';
-    win = env.win;
-    win.document.documentElement.className = 'top i-amphtml-singledoc';
-    child = win.document.createElement('div');
-    child.style.width = '200px';
-    child.style.height = '300px';
-    child.textContent = 'test';
-    win.document.body.appendChild(child);
-
-    const styleEl = win.document.createElement('style');
-    styleEl.textContent = `
-        .flexed {display: flex}
-        @media (min-width: 150px) {
-          body {display: table}
-        }
-      `;
-    win.document.head.appendChild(styleEl);
-
-    installDocService(win, /* isSingleDoc */ true);
-    installDocumentStateService(win);
-    installVsyncService(win);
-    vsync = Services.vsyncFor(win);
-    binding = new ViewportBindingIosEmbedShadowRoot_(win);
-    binding.connect();
-  });
-
-  it('should NOT require fixed layer transferring', () => {
-    expect(binding.requiresFixedLayerTransfer()).to.be.true;
-  });
-
-  it('should start w/o overscroll and set it on doc ready', () => {
-    const root = binding.getScrollingElement();
-    expect(root).to.not.have.class('i-amphtml-ios-overscroll');
-    return whenDocumentReady(win.document).then(() => {
-      expect(root).to.have.class('i-amphtml-ios-overscroll');
+  // Can only test when SD is supported.
+  describe.configure().ifChrome('EmbedShadowRoot_', function() {
```

Comment:
Safari also has `attachShadow`. Also, it's a Safari specific test suite. 😉

Reply:
Done.